Node.js/boilerplate.js

```diff
@@ -174,7 +174,12 @@ class MQBoilerPlate {
       }
 
       debug_info('Putting Message on Queue in mode ', mode);
-      mq.Put(queue, mqmd, pmo, msg, function(err) {
+      let putCall = mq.Put; 
```

Comment:
Add comment as to why this is necessary.

Reply:
Needs rewording. As is it implies that the requester app is why the call needs to be sync, which is incorrect. 

The issue lies with the responder. This call is happening in the already async responder get callback. Adding another async on top of the async callback causes 2500. That is why this needs to be a sync put.